packages/tests/README.md

```diff
@@ -0,0 +1,32 @@
+# Description
+
+This package contains tests for the `js-waku` library.
+Some of the tests are integration tests that require a running an `nwaku` or a `go-waku` node. These nodes are setup to be run in a docker container.
+
+# Running interop tests
+
+- The tests by default run against an `nwaku` with the image name specified in `nwaku.ts` with the variable name `NWAKU_VERSION`. The tests can be run against a different version of `nwaku` by setting the environment variable `NWAKU_VERSION` to the desired version.
+
+- The tests can be run against a `go-waku` node by setting the environment variable `IS_GO_WAKU` to `true`. The tests will then run against a `go-waku` node with the image name tagged with `latest`.
+
+> Please be sure to `pull` the image you intend to run the tests against before running the tests.
+>
+> You can run the tests by running the following command:
+>
+> ```bash
+> docker pull image-name
+> ```
+
+- You can also run the tests against a local `nwaku` or `go-waku` node by setting the environment variable `LOCAL_IMAGE_NAME` to the name of the image. The tests will then run against the local image.
+
+> To run the tests against a local `nwaku` or `go-waku` node, you need to build the image first. You can build the image by running the following command:
+>
+>     ```bash
+>     docker build path-to-dockerfile -t image-name
+>     ```
+>
+>     Then, you can run the tests by running the following command:
+>
+>     ```bash
+>     LOCAL_IMAGE_NAME=image-name npm run test
+>     ```
```

Comment:
Why are you doing code blocks (```) in quote blocks (>)?

Reply:
Does not...

![image](https://user-images.githubusercontent.com/110212804/228718373-03080e21-c85c-4140-b355-db9550a8ed73.png)